fastavro/_schema_public.py

```diff
@@ -0,0 +1,27 @@
+PRIMITIVES = set([
```

Comment:
What do you think about changing this file name to `_schema_common.py`? I think that might better describe that this file is common to both the pyx and py schema files.

Reply:
That sounds like a better name. I'll change it.

Reply:
Done.